index.js

```diff
@@ -1 +1,21 @@
 // phew… not a lot going on here. Please add some code!
+const bookmark = document.querySelector('[data-js="bookmark"]');
+
+bookmark.addEventListener("click", () => {
+  bookmark.classList.toggle("bookmark--active");
+});
+
+/* const answer = document.querySelector('data-js="card__answer');
+const answerButton = document.querySelector('data-js="');
+ */
```

Comment:
we dont merge unused code ;)

Reply:
oops, still have to get used to the git workflow...